### ISSUE TITLE
running container after pulling image

### ISSUE THREAD

**Issue by njm2112** [mdouchement/docker-zoom-us#9]
Install instructions are somewhat unclear... I am used to using `docker-compose` but there is no `docker-compose.yml` in this repo.

After pulling the `:latest` image, is that when you run the `install` scripts? Or do you have to first build the container?

I tried running using `docker run --name=zoom zoom-us` but Docker Desktop for Mac is reporting `Exit Code 0` -- the container stops the moment after it starts.

Also, `zoom` at the shell does not work because there's nothing aliased to the command...

Can you provide a little more info about installation/building the container, and then running it? Happy to provide a pull request with an updated `readme.md` if you can help me get this up and running.

**Comment by mdouchement** [comment]
The installation process puts this script https://github.com/mdouchement/docker-zoom-us/blob/master/scripts/zoom-us-wrapper in the host folder `/usr/local/bin` (using docker option: `--volume /usr/local/bin:/target`).
`docker-compose` lacks of flexibility to run this image.

This project is made to run on GNU/Linux as it uses on X11 an PulseAudio. It mounts host's devices in the container like `/dev/video*` and `/dev/dri` that does not exist on macOS.

**Comment by njm2112** [comment]
Thanks for clarifying this. As I mentioned, attempting to execute `zoom` at the shell prompt fails because it doesn't appear there is anything aliased to the `zoom` command (even after the installation process puts the script in the host folder). Can I manually alias the script? If so, what am I to use `alias zoom=/usr/local/bin/zoom-us-wrapper`?

Also, it just occurred to me that X11 is 32-bit and I am using macOS 10.15 (Catalina) which does not run 32-bit applications. Does this mean that I won't be able to use this without using an OS that supports launching X11?

Thanks again!

**Comment by mdouchement** [comment]
Yes you can put the script where you want on your host and make an alias.
(A symbolic link is normally done during the installation https://github.com/mdouchement/docker-zoom-us/blob/master/entrypoint.sh#L13).

I don't know about Catalina issue.
Maybe Docker for Mac should handle 32bits Linux binaries as it is a xhyve VM.
You'll need tools like XQuartz and PulseAudio on your mac.
You'll also need to find how to forward host devices (webcam, audio, etc.) to the xhyve VM.

**Comment by jonathancross** [comment]
@njm2112 any luck?

**Comment by changalberto** [comment]
The wrapper script is not being installed on Catalina even when you `sudo` I think is because of security. But when I install the script into my `~/Downloads` folder, it prompts me to give it folder access and installs successfully.

After installing the script into `Downloads` folder, I tried executing the script from there and I get the following error:

```
./zoom-us-wrapper: line 15: xdg-user-dir: command not found
./zoom-us-wrapper: line 19: xdg-user-dir: command not found
Cleaning up stopped zoom-us instances...
sudo: docker: command not found
./zoom-us-wrapper: line 57: xauth: command not found
./zoom-us-wrapper: line 57: xauth: command not found
```

**Comment by asgmeonerandom** [comment]
any update please?

**Comment by LoSunny** [comment]
+1
